code/graphics/2d.cpp

```diff
@@ -2124,6 +2125,7 @@ void gr_flip()
 	// m!m avoid running CHA_ONFRAME when the "Quit mission" popup is shown. See mantis 2446 for reference
 	if (!quit_mission_popup_shown)
 	{
+		tracepoint(fs2open, LUA_On_Frame__begin);
```

Comment:
As I said before on IRC, the two profiling systems should be combined since it's a bit cumbersome to maintain two different kinds of tracing systems if they don't share the same external API.